src/components/PageTemplate/Navbar.jsx

```diff
@@ -14,79 +14,95 @@ import {
-            <Flex flex={1} h={16} p={4} align="center" maxW="container.xl">
+const Logo = () => (
+    // TODO: update logo
+    <Link to="/"><Button fontSize={24} fontWeight={200} m={MARGIN}>InvestoBull</Button></Link>
+)
 
-                {/*Hamburger Menu, visible when screen width narrow*/}
-                <Flex display={['flex', 'flex', 'none', 'none']}>
-                    <Menu>
-                        <MenuButton as={Button} bg="transparent" pl={1} pr={1} m={MARGIN}>
-                            <HamburgerIcon w={6} h={6}/>
-                        </MenuButton>
-                        <MenuList bg={bgColor}>
-                            <Link to="/"><MenuItem>Home</MenuItem></Link>
-                            <Link to="/watchlist"><MenuItem>Watchlist</MenuItem></Link>
-                            <Link to="/news"><MenuItem>News</MenuItem></Link>
-                            <Link to="/about"><MenuItem>About</MenuItem></Link>
-                        </MenuList>
-                    </Menu>
-                </Flex>
+const HamburgerMenu = ({bgColor}) => (
+    // visible when screen width narrow
+    <Flex display={['flex', 'flex', 'none', 'none']}>
+        <Menu>
+            <MenuButton as={Button} bg="transparent" px={PADDING} m={MARGIN}>
+                <HamburgerIcon w={ICON_SIZE} h={ICON_SIZE}/>
+            </MenuButton>
+            <MenuList bg={bgColor}>
+                <Link to="/"><MenuItem>Home</MenuItem></Link>
+                <Link to="/watchlist"><MenuItem>Watchlist</MenuItem></Link>
+                <Link to="/news"><MenuItem>News</MenuItem></Link>
+                <Link to="/about"><MenuItem>About</MenuItem></Link>
+            </MenuList>
+        </Menu>
+    </Flex>
+)
 
-                {/*TODO InvestoBull Logo*/}
-                <Link to="/"><Button fontSize={24} fontWeight={200} m={MARGIN}>InvestoBull</Button></Link>
+const Links = () => (
+    // visible when screen width wide
+    <Flex display={['none', 'none', 'flex', 'flex']}>
+        <Link to="/"><Button bg="transparent" m={MARGIN}>Home</Button></Link>
+        <Link to="/watchlist"><Button bg="transparent" m={MARGIN}>Watchlist</Button></Link>
+        <Link to="/news"><Button bg="transparent" m={MARGIN}>News</Button></Link>
+        <Link to="/about"><Button bg="transparent" m={MARGIN}>About</Button></Link>
+    </Flex>
+)
 
-                {/*Button Links, visible when screen width wide*/}
-                <Flex display={['none', 'none', 'flex', 'flex']}>
-                    <Link to="/"><Button bg="transparent" m={MARGIN}>Home</Button></Link>
-                    <Link to="/watchlist"><Button bg="transparent" m={MARGIN}>Watchlist</Button></Link>
-                    <Link to="/news"><Button bg="transparent" m={MARGIN}>News</Button></Link>
-                    <Link to="/about"><Button bg="transparent" m={MARGIN}>About</Button></Link>
-                </Flex>
+const ThemeSwitch = ({tcm}) => (
```

Comment:
1. Switch disappears when I decrease my window width, but we have more than enough space to keep the switch visible

<img width="426" alt="Screen Shot 2021-06-11 at 13 41 54" src="https://user-images.githubusercontent.com/51832617/121746483-cb642d80-caba-11eb-8de1-f6cf57ebda5a.png">

Reply:
@paulsgh 
(1) I removed the switch when reaching a "small" size on purpose, as some small screens like iphone 5s will have overbleeding navbar contents if I don't:
<img width="330" alt="Screen Shot 2021-06-11 at 7 36 42 PM" src="https://user-images.githubusercontent.com/43733599/121762674-5b6e9b00-caec-11eb-804d-022f72b848e7.png">

(2a) Currently it is on==dark, off==light, and uses system settings for default
(2b) Yes I can add that

Reply:
1) got it, thanks
2) i think there's a bug then. my system is set to dark mode, and as expected, the site is set to dark mode by default — but the issue is that the switch is off. so for me on==light, off=dark
3) thanks!

Reply:
I fixed it by putting the theme toggle in the User Menu (avatar button)